system_metrics_collector/CMakeLists.txt

```diff
@@ -28,15 +28,24 @@ find_package(rclcpp REQUIRED)
 find_package(rcutils REQUIRED)
 find_package(metrics_statistics_msgs REQUIRED)
 
+include_directories(include)
```

Comment:
Is this package intended to be a C++ library? If it isn't, you should put your headers in src/

Reply:
At some point we will want to do so, but moving to src for now while we develop.